mappings/net/minecraft/server/PlayerManager.mapping

```diff
@@ -0,0 +1,121 @@
+CLASS net/minecraft/class_166 net/minecraft/server/PlayerManager
+	FIELD field_577 LOGGER Ljava/util/logging/Logger;
+	FIELD field_578 players Ljava/util/List;
+	FIELD field_579 server Lnet/minecraft/server/MinecraftServer;
+	FIELD field_580 chunkMaps [Lnet/minecraft/class_167;
+	FIELD field_581 maxPlayerCount I
+	FIELD field_582 bannedPlayers Ljava/util/Set;
+	FIELD field_583 bannedIps Ljava/util/Set;
+	FIELD field_584 ops Ljava/util/Set;
+	FIELD field_585 whitelist Ljava/util/Set;
+	FIELD field_586 BANNED_PLAYERS_FILE Ljava/io/File;
+	FIELD field_587 BANNED_IPS_FILE Ljava/io/File;
+	FIELD field_588 OPERATORS_FILE Ljava/io/File;
+	FIELD field_589 WHITELIST_FILE Ljava/io/File;
+	FIELD field_590 playerSaveHandler Lnet/minecraft/class_168;
+	FIELD field_591 whitelistEnabled Z
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;)V
+		ARG 1 server
+	METHOD method_549 getBlockViewDistance ()I
+	METHOD method_550 sendPacketAtPos (DDDDILnet/minecraft/class_169;)V
+		COMMENT Send packet to all players in the specified range of the given position
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+		ARG 7 range
+		ARG 9 dimensionId
+		ARG 10 packet
+	METHOD method_551 getChunkMap (I)Lnet/minecraft/class_167;
+		ARG 1 dimensionId
+	METHOD method_552 markBlockForUpdate (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 dimensionId
+	METHOD method_553 updateBlockEntity (IIILnet/minecraft/class_55;)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 blockentity
+	METHOD method_554 updatePlayerAfterDimensionChange (Lnet/minecraft/class_69;)V
+		ARG 1 player
+	METHOD method_555 respawnPlayerEntity (Lnet/minecraft/class_69;I)Lnet/minecraft/class_69;
+		ARG 1 player
+		ARG 2 dimensionId
+	METHOD method_556 sendTimeWeatherPacket (Lnet/minecraft/class_69;Lnet/minecraft/class_73;)V
+		ARG 1 player
+		ARG 2 world
+	METHOD method_557 connectPlayer (Lnet/minecraft/class_10;Ljava/lang/String;)Lnet/minecraft/class_69;
+		ARG 1 loginNetworkHandler
+		ARG 2 name
+	METHOD method_558 sendPacketAroundPlayer (Lnet/minecraft/class_54;DDDDILnet/minecraft/class_169;)V
```

Comment:
[sendToAround](https://maven.fabricmc.net/docs/yarn-1.21+build.6/net/minecraft/server/PlayerManager.html#sendToAround(net.minecraft.entity.player.PlayerEntity,double,double,double,double,net.minecraft.registry.RegistryKey,net.minecraft.network.packet.Packet)) in Yarn